qa/suites/fs/basic_functional/tasks/acls-fuse-client.yaml

```diff
@@ -0,0 +1,8 @@
+tasks:
+  - install:
```

Comment:
Oops, this doesn't work; my fault for recommending it. The install task does a full "unwind" even if we are only really trying to install system packages. This means that once the test finishes the install task will remove all of the packages.

Let's just move the packages to `qa/cephfs/begin.yaml`. I don't think there is an easier way to do it without changing a lot of code.

Reply:
Shouldn't the natural order for teardown be to stop anything "Ceph" first and then remove related files and then uninstall packages?

Reply:
> Let's just move the packages to qa/cephfs/begin.yaml. I don't think there is an easier way to do it without changing a lot of code.

Made the change.

Reply:
Was this in context to my last comment? I don't get the context, otherwise.

Reply:
> Shouldn't the natural order for teardown be to stop anything "Ceph" first and then remove related files and then uninstall packages?

Yes but you now have two `install` tasks:

````
  - install:
      extra_packages:
        deb:
        - python3-cephfs
        - cephfs-shell
        rpm:
        - python3-cephfs
      extra_system_packages:
        deb:
        - bison
        - flex
        - libelf-dev
        - libssl-dev
        rpm:
        - bison
        - flex
        - elfutils-libelf-devel
        - openssl-devel
  - ceph: null
  - ceph-fuse: null
  - install:
      extra_system_packages:
        deb:
        - libaio-dev
        rpm:
        - dump
        - indent
        - libaio-devel
        - btrfs-progs-devel
````

From: /ceph/teuthology-archive/rishabh-2019-10-04_12:43:36-fs-add-test-for-acls-distro-basic-smithi/4357983/teuthology.log

the second install task runs after the `ceph` task.